justfile

```diff
@@ -68,11 +68,11 @@ check *args:
 
 # Build for a Linux target (no SGX)
 build-no-sgx *args:
-  cargo build {{args}}
+  cargo build --target x86_64-unknown-linux-gnu {{args}}
 
 # Run on a Linux target (no SGX)
 run-no-sgx *args:
-  cargo run {{args}}
+  cargo run --target x86_64-unknown-linux-gnu {{args}}
```

Comment:
Don't need that. The local target is the default.

Reply:
It is actually needed because the rust-toolchain.toml does specify the fortanix toolchain, I am simply forcing to use the original linux toolchain

Reply:
I kind of get why this can be confusing, but default toolchain != default target. It works perfectly on my end without the "--target x86_64-unknown-linux-gnu". I'd be curious to know in what configuration you were for sgx to be the default target.